### DISCUSSION TITLE
5.x的分面图color设置连续型，但是图例却是离散的，4.x的图例是连续的，5.x以后会修改成和4.x的一致吗

### DISCUSSION THREAD

**Discussion by xinxin12138** [antvis/G2#5136] [Q&A] [answered]
![23337ecbf0512998b1e03d9e1c0bab5](https://github.com/antvis/G2/assets/56461371/be1407c5-a7ca-478d-94af-704ba6914913)

**Comment by pearmini** [comment]
这个问题已经在修复中了。

**Reply by pearmini** [reply, marked as answer]
这个 PR 修复了这个能力：https://github.com/antvis/G2/pull/5137
